boards/nucleo-l552ze-q/doc.txt

```diff
@@ -34,6 +36,18 @@ and debug via GDB by simply typing
 make BOARD=nucleo-l552ze-q debug
 ```
 
+### Flashing the Board Using ST-LINK Removable Media
+
+On-board ST-LINK programmer provides via composite USB device removable media.
+Copying the HEX file causes reprogramming of the board. This task
+could be performed manually; however, the cpy2remed (copy to removable
+media) PROGRAMMER script does this automatically. To program board in
+this manner, use the command:
+```
+make BOARD=nucleo-l552ze-q PROGRAMMER=cpy2remed flash
+```
+
+
```

Comment:
Note that all Nucelos support this, provided the firmware of the ST-Link is up to date. Maybe add a line to point out that updating the ST-Link firmware could be required? (ST actually provides a .jar file to perform the update, and surprisingly Java actually lived up to the "write once run everywhere" and it does actually work also on Linux.) Maybe the link to that website can be added here?

Reply:
even older boards? I'll try!

Reply:
I could add appropriate text to the documentation page (doc.txt file) but I have only one Nucleo board. My board have such firmware:

``` 
Version: V2J37M26
Build:   May 13 2020 17:07:43
````
Which probably states that this is version 2.37.26. Looking at the [STM update page](https://www.st.com/en/development-tools/stsw-link007.html) this is third most recent firmware. 

Do you think about note. "This PROGRAMMER was tested using 2.37.26 version of the ST-LINK firmware. You could find firmware update at [STM webpage](https://www.st.com/en/development-tools/stsw-link007.html)."

Reply:
I added note to the doc.txt concerning ST-LINK firmware and fix small issue with ```}``` in ```cpy2remed.sh``` script.